### DISCUSSION TITLE
node-red update script doesn't update Node.js

### DISCUSSION THREAD

**Discussion by TychoWerner** [tteck/Proxmox#3246] [Help Needed] [answered]
Hi, recently node-red came out with the v4.0.0, which requires Node.js v18 or later.
The current update script doesn't update this, so it is stuck on v16.20.2 in my case.
Running the general LXC updater does also not update this

Running **npm install -g n** and **n stable** fixed my issue
Hope the appropriate commands can be used in the node-red updater

**Comment by tteck** [comment, marked as answer]
The script installs Node v20. I don't recall the script ever using v16.
Maybe at the very beginning  🤷🏻‍♂️

**Reply by TychoWerner** [reply]
It's already a really old install, if oldest files to be believed: 2022-11-17
(ls -lact --full-time /etc |tail)

**Reply by TychoWerner** [reply]
Haha nice edit, I guess it won't be a probably then for a lot of people?
You can close it if you think it's not an issue
